schema.go

```diff
@@ -101,13 +123,9 @@ func (j *JSONSchema) ScheYAML(cfg *Config) *yaml.Node {
 		case j.Default != nil:
 			result.Value = fmt.Sprint(j.Default)
 
-		case j.Type == TypeString:
-			result.LineComment = cfg.TODOComment
-			result.SetString("")
-
 		default:
 			result.LineComment = cfg.TODOComment
-			result.Value = j.Type.DefaultValue()
```

Comment:
In that case, that method can be removed entirely :D

Reply:
Removed